shopgate.php

```diff
@@ -737,39 +788,41 @@ public function getContent()
 	{
 		include_once dirname(__FILE__).'/backward_compatibility/backward.php';
 
-		$output = '';
+		$output         = '';
 		$shopgateConfig = new ShopgateConfigPresta(
-			Configuration::get('SHOPGATE_CONFIG') ?
+			is_array(unserialize(Configuration::get('SHOPGATE_CONFIG'))) ?
 				unserialize(Configuration::get('SHOPGATE_CONFIG')) :
-				array()
+				array ()
```

Comment:
Here again, please store the result in a temporary variable instead of unserializing two times